nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-client-dto/pom.xml

```diff
@@ -26,5 +26,13 @@
             <groupId>com.wordnik</groupId>
             <artifactId>swagger-annotations</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-utils</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-api</artifactId>
+        </dependency>
```

Comment:
Are these dependencies necessary? Would like to have this artifact as 'light' and 'portability' as possible. Should just be definitions of transport objects for using the REST API.